AEPCore/Sources/migration/V4MigrationConstants.swift

```diff
@@ -50,8 +50,8 @@ enum V4MigrationConstants {
         static let V4_PUSH_TOKEN = "ADBMOBILE_KEY_PUSH_TOKEN"
 
         enum DataStoreKeys {
-            static let IDENTITY_PROPERTIES = "identitiesproperties"
-            static let PUSH_ENABLED = "ADOBEMOBILE_PUSH_ENABLED"
+            static let IDENTITY_PROPERTIES = "identity.properties"
```

Comment:
Looks like we forgot to update these once we decided on a format in the style guide for data store keys.

@shalehaha we may need to update the other keys as well.